Add release command and documentation (#87)
philips-labs/slsa-provenance-action: release.md

# Automated release procedure

> A make script has been created to automate the manual procedure.

Reviewer: ```suggestion
To make a new release you can make use of the following `make` task.
```

> Execute the following command:

Reviewer: ```suggestion

```

```bash
make gh-release NEW_VERSION=v0.6.0 OLD_VERSION=v0.5.0 DESCRIPTION="A test release to see how it works"
```

> `NEW_VERSION` is the version that you want to release.

Reviewer: ```suggestion
`NEW_VERSION` the version that you want to release.
```

> `OLD_VERSION` is the previous version you wish to overwrite in the markdown and yaml files.

Reviewer: ```suggestion
`OLD_VERSION` the current version you wish to replace in the markdown and yaml files.
```

> `DESCRIPTION` is the description to use in the annotation of the tag and commit description.

Reviewer: ```suggestion
`DESCRIPTION` the annotation used when tagging the release.
```
